orc8r/lib/go/registry/interceptors.go

```diff
@@ -45,5 +52,24 @@ func TimeoutInterceptor(ctx context.Context, method string, req, resp interface{
 func CloudClientTimeoutInterceptor(ctx context.Context, method string, req, resp interface{},
 	cc *grpc.ClientConn, invoker grpc.UnaryInvoker, opts ...grpc.CallOption) error {
 
-	return TimeoutInterceptor(unary.OutgoingCloudClientCtx(ctx), method, req, resp, cc, invoker, opts...)
+	return TimeoutInterceptor(OutgoingCloudClientCtx(ctx), method, req, resp, cc, invoker, opts...)
+}
+
+// OutgoingCloudClientCtx amends outgoing cloud client context with magic client CSN metadata
+// if the original context doesn't already have client certificate serial number metadata
+func OutgoingCloudClientCtx(ctx context.Context) context.Context {
```

Comment:
```suggestion
func outgoingCloudClientCtx(ctx context.Context) context.Context {
```
I think this could be private now.

Reply:
Good point, thanks! See https://github.com/magma/magma/pull/12744/commits/060e5b323367812baee3f07f701b825d4ed36274